src/components/layout/playground/applyForm.tsx

```diff
@@ -327,13 +327,13 @@ const MainForm: React.FC<RequestProps> = ({ request }) => {
             className="w-full"
             error={errors.linkedinUrl?.message}
             {...myRegister('linkedinUrl')}
-            placeholder="LinkedIn username"
+            placeholder="linkedin.com/in/"
           >
             LinkedIn
           </TextInput>
         </div>
         <div className="flex flex-row justify-start gap-5">
-          <div>Have you applied for a Playground role in the past?</div>
+          <div>Have you applied for a Playground volunteer role in the past?</div>
```

Comment:
Thanks for the changes! I really like them all, but I have a question about this specific one. There are paid and volunteer roles, so I'm not sure this is the best phrasing 🤔

Reply:
Thanks for clarifying, updated!